test-series/1/test-1.1.0.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
<p>Isn't this already provided by <code>astropy.cosmology.Planck15</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
As Steven mentioned in the 1.0.0 PR, this is ever so slightly different and was used in the original.